docs/roadmap.md

```diff
@@ -0,0 +1,169 @@
+# Roadmap
+
+* [**Simple installation**](#simple-installation): importing a project should be
+  simple and require as few steps as possible.
+* [**Correct diagnostics**](#correct-diagnostics): the editor should never show
+  spurious red squiggles, even at the cost of higher latency.
```

Comment:
Never as in "never" or "much more rarely than the others"?

Reply:
s/never/not/